tools/release_do_nothing.py

```diff
@@ -10,879 +10,869 @@
-        "- Requirements: align the packages and pins with those in the Iris repo\n"
-        "- Maintainers: update with any changes to the dev team\n"
-        "- MAKE SURE everything else is correct - plenty of other things "
-        "might need one-off changes.\n"
-    )
-    _wait_for_done(message)
-
-    # TODO: automate
-    message = (
-        "No other file normally needs changing in iris-feedstock, so push up "
-        "the changes to prepare for a Pull Request:\n"
-        f'git commit -am "Recipe updates for {release_strings.tag} .";\n'
-        f"git push -u origin {release_strings.tag};"
-    )
-    _wait_for_done(message)
+            message = (
+                f"Is the latest commit on {rc_branch} over 1 month ago?"
+            )
+            archive_rc = None
+            while archive_rc is None:
+                valid_entries = ["y", "n"]
+                age_check = self.get_input(message, " / ".join(valid_entries))
+                match = [age_check.casefold() == e.casefold() for e in valid_entries]
+                if not any(match):
+                    self.report_problem("Invalid entry. Please try again ...")
+                else:
+                    archive_rc = match[0]
+
+            if archive_rc:
+                # We chose this odd handling of release candidate branches because
+                #  a persistent branch will gradually diverge as `main` receives
+                #  automatic and manual maintenance (where recreating these on
+                #  another branch is often beyond Iris dev expertise). Advised
+                #  practice from conda-forge is also liable to evolve over time.
+                #  Since there is no benefit to a continuous Git history on the
+                #  release candidate branch, the simplest way to keep it  aligned
+                #  with best practice is to regularly create a fresh branch from
+                #  `main`.
+
+                date_string = datetime.today().strftime("%Y%m%d")
+                message = (
+                    f"Archive the {rc_branch} branch by appending _"
+                    f"{date_string} "
+                    "to its name.\n"
+                    f"e.g. rc_{date_string}\n\n"
+                    f"({__file__} includes an explanation of this in the "
+                    f"comments)."
+                )
+                self.wait_for_done(message)
 
-    message = (
-        f"Follow the Pull Request process to get {release_strings.tag} branch "
-        f"merged into upstream/{upstream_branch} .\n"
-        "Specific conda-forge guidance will be automatically given once the "
-        "PR is created."
-    )
-    _wait_for_done(message)
+                message = (
+                    "Follow the latest conda-forge guidance for creating a new "
+                    "release candidate branch from the `main` branch:\n"
+                    "https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds\n\n"
```

Comment:
Need to include a note to check that conda-forge automation has updated the feedstock `README` to reference the release candidate channel - this is considered evidence that things have worked correctly.

Reply:
8577b7914d251593bde0473e4d09ce493393120a